tests/Melodiia/Bridge/Doctrine/DoctrineDataStoreTest.php

```diff
@@ -22,8 +23,10 @@ public function testItSaveUsingTheEntityManager()
         $entityManager = $this->prophesize(EntityManagerInterface::class);
         $entityManager->persist($obj)->shouldBeCalled();
         $entityManager->flush($obj)->shouldBeCalled();
+        $managerRegistry = $this->prophesize(ManagerRegistry::class);
+        $managerRegistry->getManager()->willReturn($entityManager->reveal());
 
-        $datastore = new DoctrineDataStore($entityManager->reveal());
+        $datastore = new DoctrineDataStore($managerRegistry->reveal());
         $datastore->save($obj);
     }
```

Comment:
Missing test for the `getPaginated` method :)

Reply:
Fixed